testing/python/fixture.py

```diff
@@ -992,8 +992,8 @@ def test_add(arg2):
         result.stdout.fnmatch_lines(
             [
                 "*ScopeMismatch*involved factories*",
-                "* def arg2*",
-                "* def arg1*",
+                "test_receives_funcargs_scope_mismatch.py:6:  def arg2(arg1)",
+                "test_receives_funcargs_scope_mismatch.py:2:  def arg1()",
```

Comment:
Hmm strange, `def arg1` to me looks like in line 3, and `def arg2` on line 7... 🤔

Reply:
It is referring to the decorator.
I've just tried with py38 again (given @asottile's comment), but there it is also line 6 and 2.
I think this is fine, since the error comes from the `@fixture` after all.
(But it refers to the first fixture)
The -1 is coming from https://github.com/vim/vim/commit/461a7fcfce3cd6414f990037e6468af3b5ccf119#diff-de9ab8906fef668ed8042046e0a1eb02L8145 btw.